lib/ClimaCoreTempestRemap/src/exodus.jl

```diff
@@ -68,6 +94,7 @@ function write_exodus(
         defDim(dts, "num_el_in_blk1", num_el_in_blk1)
         defDim(dts, "num_nod_per_el1", num_nod_per_el1)
         defDim(dts, "num_att_in_blk1", num_att_in_blk1)
+        defDim(dts, "nvertices", nvertices)
```

Comment:
this should be the same as `num_nodes` (we should keep the naming consistent with what we have at the moment)